app/views/EPaye/ineligible/IneligibleTemplatePage.scala.html

```diff
@@ -52,17 +53,17 @@
     </p>
 }
 
-@layout(pageTitle = Some(Messages.NotEligible.`Call us`.show)) {
+@layout(pageTitle = Some(pageh1.show)) {
     <h1 class="govuk-heading-xl">
     @{
-        Messages.NotEligible.`Call us`.show
+        pageh1.show
     }
     </h1>
     @leadingContent
     <div class="common-eligibility">
         <p class="govuk-body">
         @{
-            Html(Messages.NotEligible.`For further support you can contact the Payment Support Service on 0300 200 3835 to speak to an advisor.`show)
+            Html(Messages.NotEligible.`For further support you can contact the Payment Support Service on 0300 200 3835 to speak to an advisor.` show)
         }
```

Comment:
looks like this is still missing a dot...

Reply:
woops, great catch! will fix